### ISSUE TITLE
FNX-500 ⁃ [Bug] More options button claims 4 items when there are only 3

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#2143]
## Steps to reproduce
Enable talkback
swipe to the More options menu
double tap to open the More options menu so first item Settings is selected

### Actual behavior
Talkback reads out "Settings, in list of 4 items" 

### Device information
Talkback should read "Settings, in list of 3 items" since there are only three items (plus a separator which should not be counted.

* Android device: ? Samsung Galaxy S6
* Fenix version: ? Firefox Preview 1.0.1918 (Build #11181249 68.0a1-20190422094240)

**Comment by kbrosnan** [comment]
What do you mean by the more options menu?

**Comment by asadotzler** [comment]
> 
> 
> What do you mean by the more options menu?

The three vertical dot menu next to the private browsing button at the top right of the home screen.

**Comment by mihai-adrian** [comment]
Before working on a solution, I have a question: is the divider necessary? There are only 3 items in menu, and it is not consistent with the menu from the bottom that does not have any divider.

**Comment by mcarare** [comment]
@asadotzler, @eeejay Do you think it will be better to not announce the list count at all? Overriding list count requires too much overhead. Thank you!

**Comment by eeejay** [comment]
No, I think the list count is needed. I'm trying to remember if I attempted to fix this myself and ran into it being too hard or not. I'll take a look at this soon.
